wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -555,21 +556,31 @@ private RecipeParser getRecipeParser(StageContext context)
     @Nullable
     private final String onError;
 
+    // this is a hidden property which preserves some sort of 6.3 backward compatibility,
+    // if any user wants the "send-to-error-port" default, they can use this flag, but it won't be shown in UI.
+    @Nullable
+    private final Boolean keepOldErrorDefault;
```

Comment:
Would it be better to make this an instance-level configuration? This forces users to always have to modify existing pipelines in order to enable this behavior.

Reply:
I agree. We should do it from runtime properties. Having it as a config does not solve a problem as user have to change every pipeline. If they go change every pipeline, they can set onError  directly. And runtime properly is something they can set on namespace/instance level

Reply:
Also, does it syncs up the UI? We need to ensure backend default and UI default are in sync

Reply:
Added the config as instance level. 
It won't sync with UI, the UI default is inside the widget file, like this, https://github.com/data-integrations/wrangler/pull/492/files#diff-6b579e1ba0e29fd41b29cceb044c8227f2c4491be4bedf3410b4945fd3425055R58